tests/unit/apps/test_channel_based.py

```diff
@@ -59,6 +59,54 @@ def test_application_versionless(model):
     assert app.get_latest_os_version(units["glance-simplestreams-sync/0"]) == app.channel_codename
 
 
+def test_channel_based_application_latest_stable(model):
```

Comment:
The test that calculates the overall cloud version is on the analyze test. This test is showing that the channel based applications are totally dependent on the target passed because the application might say that is ussuri, but it will crossgrade to the `N-1` release from the target and then change to the target channel